config/initializers/devise.rb

```diff
@@ -10,6 +10,7 @@
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
 Devise.setup do |config|
+  config.secret_key = ENV.fetch('DEVISE_SECRET_KEY', 'd6cfe9c078c3618e720cb9f3c1579ac77202a9364bca663aebf03758b992a5a986197de946e373e1d8aca365d7a5531e22902e6414874a895c50aeb76f3c9f08')
```

Comment:
@dougmartin The build docker image process was failing because there was no `DEVISE_SECRET_KEY` value defined here. It wasn't clear to us how to define it. We did try adding it to the Actions Secrets and Variables section of the Settings but that didn't seem to work. The [Portal's config/initializers/devise.rb file uses a fallback like this](https://github.com/concord-consortium/rigse/blob/master/rails/config/initializers/devise.rb), so we went with that and once we did the build docker image process worked. Is there maybe a better option?